include/common.h

```diff
@@ -927,7 +935,7 @@ extern struct EffectObj effect_objects[0x20];
 extern struct ItemObj item_objects[0x20];
 extern struct MiscObj misc_objects[0x40];
 extern struct LayerObj layer_objects[4];
-extern struct QuxObj qux_objects[1];
+extern struct QuxObj qux_object;
```

Comment:
The build error is happening since this didn't get the symbol renamed I think

Reply:
yeah sorry I just saw this — I renamed it to singular to match `engine_obj` and `bar_object` since they all seem to be singletons